doc/conf.py

```diff
@@ -133,6 +133,18 @@
 # Plot directive configuration
 # ----------------------------
 
+plot_pre_code = '''
+# Default imports from plot_directive sphinx extension.
+import numpy as np
+from matplotlib import pyplot as plt
+
+# Initialize random number generator for consistent plots.
+import random
+random.seed(20021210)
+del random
```

Comment:
what's the point of importing random, then deleting random, then using np.random?